external/get_modules.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
```

Comment:
same for licence

Reply:
Ok, updated "NVIDIA" to "NVIDIA CORPORATION".